tests/forms_tests/field_tests/test_datetimefield.py

```diff
@@ -58,12 +55,33 @@ def test_datetimefield_clean(self):
                 ' 2014-09-23T22:34:41.614804 ',
                 datetime(2014, 9, 23, 22, 34, 41, 614804),
             ),
+            (' 2014-09-23T22:34Z ', datetime(2014, 9, 23, 17, 34)),
+        ]
+        f = DateTimeField()
+        for value, expected_datetime in tests:
+            with self.subTest(value=value):
+                cleaned_value = f.clean(value)
+                self.assertTrue(is_naive(f.clean(cleaned_value)))
```

Comment:
Do you need to `clean` the `cleaned_value`? (same applies further down)

Reply:
No, It remained by mistake.
I am waiting for the decision about this ticket. if the solution got approved I will fix it.
Thanks